src/Core/TileMesh.js

```diff
@@ -169,7 +169,11 @@ TileMesh.prototype.center = function center() {
 };
 
 TileMesh.prototype.OBB = function OBB() {
-    return this.geometry.OBB;
+    var zFactor = this.material.uniforms.zFactor.value;
+    var OBB = this.geometry.OBB;
+        OBB.scale.z = zFactor;
```

Comment:
Code indentation is still wrong (https://github.com/iTowns/itowns/pull/511#discussion_r145419514).
And please use `const` instead of `var`.

Reply:
what wrong? it work for me!